runbotics/runbotics-orchestrator-ui/src/src-app/store/slices/CredentialCollections/CredentialCollections.slice.ts

```diff
@@ -7,7 +7,8 @@ import { CredentialCollectionsState } from './CredentialCollections.state';
 import * as credentialCollectionThunks from './CredentialCollections.thunks';
 
 const initialState: CredentialCollectionsState = {
-    credentialCollections: [],
+    allCredentialCollectionsByPage: null,
```

Comment:
I had a conversation about it during the #mvp implementation and the argument that made me change it is because initial state is actually null. Empty array can be when fetching occured but there is no data in the db. Additionaly it is more clear when I then use something like !credentialCollections rather that credentialCollections.length > 0.